### DISCUSSION TITLE
[Feature] Add time offset feature for editing date and time.

### DISCUSSION THREAD

**Discussion by MariasStory** [immich-app/immich#13946] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

In the "Edit Date and Time" window, include an option to apply a time offset based on the date of the first selected image. This adjustment would correct photo timestamps if the camera’s clock was set incorrectly.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#9430
